packages/collector/test/tracing/cloud/gcp/pubsub/test.js

```diff
@@ -203,8 +223,13 @@ mochaSuiteFn('tracing/cloud/gcp/pubsub', function () {
         GCP_PUBSUB_SUBSCRIPTION: subscriptionName
       }
     });
+
     ProcessControls.setUpHooksWithRetryTime(retryTime, publisherControls, subscriberControls);
 
+    before(async () => {
+      await agentControls.clearReceivedData();
```

Comment:
Could old received data become a problem for other tests, too? We could maybe also move this up to line 70 and make it a `beforeEach`? (Not sure though.)

Reply:
Will check 👍

Reply:
I removed the section, because the cleanup hooks are calling clearReceivedData already within `beforeEach`